### PR TITLE
Verification

### PR DESCRIPTION
**Pull request by AbstractsMachine**
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

I’m @cryptodani and my Ethereum address is 0x2Cd23B6eEA4663392c6146A34E682bb8A9C4A95B
-----BEGIN PGP SIGNATURE-----

xA0DAAoBmPhzlbDc2e8By+F0AOIAAAAA5knigJltIEBjcnlwdG9kYW5pIGFuZCBt
eSBFdGhlcmV1bSBhZGRyZXNzIGlzIDB4MkNkMjNCNmVFQTQ2NjMzOTLkYzYxNDZB
MzRFNjgyYmI4QeI5QzRB4Tk14EIAwsFcBAABCgAQBQJcuHh+CRCY+HOVsNzZ7wAA
8QYQAHX6qy3QlGXtbWd4lsMuOierfWPzeJUC6rwqNOlxNg5aG8acKU/Lsp2UrGY3
mtCYqb5p4F9L7l9bhFX9GFByDnw6B1AQ7bCcG3NPJyBkf6aPVEkdPeJ5y7G/vXr3
rjqNDAVtmm/hmX70Vq8GyzB49M5zHi1Xj6ZkJx52CdNTkTGhGXU526LlKMX+Nm2G
ihApKUrV40hEs4bmlXpd9zBPLhAIxC64ou454FJmDMsdhXV0lPV3WI9Buu4h3RMn
Cyhi78dis9JwIOrsGfOQIsCvZfisQKK3jDrMyzFX4FGR+9sy5A+Yr+7D3u14eBBU
Qm4KLW2SNpUjWZwrQN6q9fjSh8alx3/T6aCj2WAvVpFkoiMx9qxujqS0lUCuBIXv
KTb22nVF8gCBd6n9GOHTS3A5BknLWC8Iu3xRKGJfQhSEIi85+hZktKOil8/Yqd8u
K1kQzB6rPMhhFy9N98cada4mBTHlWc2EwkiiRoboFjQ+/tqlEPm4vbzRCGnbjCzz
xVKDwj/EjL7pDueko7O+RRTb4zxEu72LIXJ/mlKqxx44wjJ4YtmUS8ku96A/LieZ
LTxokl+GSWNT/NN5uOrLtb74f+Wrt8jTaMiPQaTTJB+VYNAMVKGlzl82CK4KyhFf
W6wo54h7kqURuIfCTnpIQRE0l2wCmJmWBKF/shEOQ/JlyyrE
=UBgu

-----END PGP SIGNATURE----- 
```
EDIT: wrapped in comment block to fix formatting.

### CODE REVIEW

**Comment by lkngtn** [comment]
@cryptodani Having trouble verifying your message on keybase.io/verify do you think you can try regenerating it using https://keybase.io/sign

**Comment by AbstractsMachine** [comment]
I just updated my keybase and now I can't sign anymore, the terminal just bugs and nothing comes up......
Don't know if it's a bug in my account setting in their DB or with the client itself :(

**Comment by lkngtn** [comment]
Does it work if you use the web ui https://keybase.io/sign ?

Also for the terminal are you using ` keybase pgp sign -c -m "my message" `?

**Comment by cryptodani** [comment]
Something with the characters/my keyboard in my original messages makes it bug.

New Message to verify : "do or do not there is no try"
Thanks for your patience :)
-----BEGIN PGP MESSAGE-----
Comment: https://keybase.io/download
Version: Keybase Go 3.2.2 (darwin)

xA0DAAoBmPhzlbDc2e8By+F0AOIAAAAA5GRvIG9yIGRvIG5vdCB0aGXjcmUgaXMg
bm/iIHRyeQDCwVwEAAEKABAFAly9eV0JEJj4c5Ww3NnvAAAp6RAAGBtcPhTrzW+3
HhyqvyyI1mt+jaURVOJydEpa44MKTsRodIVtT3GP2NiYtUkCO5G5zyRycRxDkwF8
Nhhg9oqn7z2pg0LmThF5VQIpUmAFY+mc97SRx/AlYhVs34ALGs0ORF2rTeCizzsf
L/NXmVrJzZed/alvm1+GhduVwPCL5szcD+D4mzDoNyPnSjgRA9VT5nkCWUAKkTwu
I30ac7PisFeBYZmNE+OxZG3YN6Q+igCEI21/MVWnkX5k+cViJZEISYQbTjLb9KtU
KEtBu7TddvPkpHpC4gho4cP3r4YMR009UYImhddkGMhvuCO/F06a4Woez/ZPOV5d
gHgpUWJdF/BQEYZjAMSchS+kU4eIADN379igMfSD+bJPy4LAWPcmrQQIynItEEhA
b/1Wz+3/td6oSNp2CueVv2KOFFvnczB5vBPzG0uebOnq0wxU/PYx0vpIdx6ar9UN
ljrk3PJFutKZPyjBcGLhqU+jUuSJRBtGXQJJOMf1bhzszevKhqfqdj3CGpD+jyg3
O0a5uFXoP6k6y8ab/7svIgl0vHsheK7HK5B1ncIVMUqK0zbsMToJACm1telZsv3+
fOj4Opv/W06zuAObRkavvZe4mT8+lR+AGvgKrnFw+5pUtybk+WFUquZ1QngPHJrL
t1K+gFe0cSL3KWZxQ1YWsvAJyb2gibQ=
=M/Yi

**Comment by cryptodani** [comment]
"my message"
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

my message
-----BEGIN PGP SIGNATURE-----

wsFcBAEBCAAQBQJcveCxCRCY+HOVsNzZ7wAAHu0QAIcKVAFVqdGYw2EFybmrEUHh
Syat/0SEnmDaoCnnapVVZWD7ag/9IcRCTyTSj2cUUui0V27TyoQ+t5UVTpskSMx6
t+MJaA4p9TVEX4AjfwgJPRRV4XlM/7g+TT8DgMrKQ8JraHZ7VHGHAoJyABwE24Xy
ubCWkKGZQJk+Zh7ErFclI4QCR/Y2C+CuNoLYe/l5mT1Tkk4LTK6e9A4U1wv9IkL9
GQoR1tXDrJe44GLt9t7HNz2SppVoxsU0l4ROMIWQ2aw8hf9HCcQ1aAa2cK257vTJ
w8+FI8VtzkrxPjrg3c6ToWeV+AZdQJXU4vR8k/JVTkQHRL/w7MdHQ9VRM60DgIm2
X2qkISqRU45mLggdI31ZsbtRH0IVU2Q54CYwOVq3jXBqF+y+64WArctRgywTzHhg
RwYji69sYfV4rVNR21XTxrO8Ppy2uAWbn6iXqlnidZVfLwF4NOia9f686tJdf6c0
GhN6o8X1llLRSOE7X89IH4OPjS4eoO/A6RSAGmcUo87c3D4pXHRFmB5nQSb/cpxM
ci5F2QZRuzu5BPxtNyd7omiF4uyJngw8sKqTPUasudhTNOCMlFUrezfLPmkFD5UD
LZjKha18Ir/akPXBmjRbLnT92RTIGiBwZh5LzuGfbKThS1ZNBLQsz5H4eDbgw3Hq
3vsjPcPyt47e0kUSx/DH
=8w0r
-----END PGP SIGNATURE-----

**Comment by AbstractsMachine** [comment]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

I’m @cryptodani and my Ethereum address is 0x2Cd23B6eEA4663392c6146A34E682bb8A9C4A95B
-----BEGIN PGP SIGNATURE-----

wsFcBAEBCAAQBQJcvegcCRCY+HOVsNzZ7wAAlsMQAH8EFy/1ZwvA1LtTrPOcAjDR
W38ps3sjRXEM2chhegHe2NuC6xNPihZmnZ3G8bJ8DsiYvcC7FMhbs3Smm5lPGTbT
h3InEG3NtEXup3GWsh8Um6aAwk6HWtytBGvzkOR81hu9SRSNdukgVmzwVqO4Os3l
wUb7P0F3NDb43tW2bjTvCNAoxXkK/Q8xueXtU+sGtmnp82SdFCvAVShgTq7ouzCA
lJrji0fDLXCuIa1Lphh3w4djGX4GE/JWCE8fdIf8ka3+vhhJ2kfCF9YdhKyPLZuR
jQUxdWHqP8+FCou5oW99zMGbDnSV4VmXFezhN5Li5PjaEDLjl49aaNPwRxiPtsWH
VCw6RRpQCGW+qf3iffIhbeeeMY2cHXpUXbnnKJ+AHq70rZ5u982jXNN1sVi+zAY/
MsHTBnHMApuVkk3WlTT82Zfv0bKwv5lEsqNAce+hJmwyq/YKXaybkCTVjXfDHlA4
Fshp2hbJqbm4AeX0sf3AMOVzW1zJ1mcGggfc3BQOI5uO9PHH2OxW+BquzRgiGln0
Kk+2DpB5JmL5EiZbfvmt+rsUH+G3tW+7ycYH6TQY8QC/4qEvH6ifEPYJpu195OKq
LnJrBidBMQPUOWp8BBqi5PoSTQEUprZsZdAnSeG+3amNN9camvHgpiOLxadSYHSC
kcaF88Pye/6bLJDRfG+C
=Lkhl
-----END PGP SIGNATURE-----

```

**Review by lkngtn — APPROVED** [review]
Verified!

**Comment by lkngtn** [comment]
@bingen @jjperezaguinaga @stellarmagnet Please approve when you have a chance. :)

File: README.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ The Aragon Cooperative DAO is located at: https://mainnet.aragon.org/#/0x940B2B5
 | [@ottog](https://keybase.io/ottog) | `0x9819c1f48397eca46e80a7aa549685ca1683b804`|
 | [@giveth](https://keybase.io/giveth) | `0x839395e20bbB182fa440d08F850E6c7A8f6F0780`|
 | [@gustavosegovia](https://keybase.io/gustavosegovia) | `0x6543c99d0e073c140Fd08A741c6cfdcd1449da94`|
-| | |
+| [@cryptodani](https://keybase.io/gustavosegovia) | `0x2Cd23B6eEA4663392c6146A34E682bb8A9C4A95B`|
```
Comment (bingen) [inline README.md]: I'm afraid there's a copy-paste minor error here ;-)

**Review by bingen — APPROVED** [review]

**Pull request merged**